wezterm-gui/src/commands.rs

```diff
@@ -1445,6 +1445,23 @@ pub fn derive_command_from_key_assignment(action: &KeyAssignment) -> Option<Comm
             menubar: &["Edit"],
             icon: Some("md_content_copy"),
         },
+        FloatPane(SpawnCommand {
+                          domain: SpawnTabDomain::CurrentPaneDomain,
+                          ..
+                      }) => CommandDef {
+            brief: label_string(action, "Create a float pane".to_string()).into(),
+            doc: "Create a float pane"
+                .into(),
+            keys: vec![(
+                Modifiers::CTRL
+                    .union(Modifiers::ALT)
+                    .union(Modifiers::SHIFT),
+                "p".into(),
+            )],
+            args: &[ArgType::ActivePane],
+            menubar: &["Shell"],
+            icon: Some("cod_split_vertical"),
```

Comment:
Awesome that you created this PR! Trying to do my part in getting this merged by going though the code and thought i would add this:

This icon should probably be changed to something resembling a floating window, my suggestion would be: `cod-primitive_square`, `nf-cod-chrome_restore` or `cod-layout_panel_center`